README.md

```diff
@@ -1 +1,23 @@
-# ipc-messanger
\ No newline at end of file
+[![Build Status](https://travis-ci.org/LuckboxGG/ipc-messenger.svg?branch=master)](https://travis-ci.org/LuckboxGG/ipc-messenger)
+
+[![Coverage Status](https://coveralls.io/repos/github/LuckboxGG/ipc-messenger/badge.svg?branch=master)](https://coveralls.io/github/LuckboxGG/ipc-messenger?branch=master)
+
+### Usage
+
+```
+const ipcMessenger = new RedisIPCMessenger({
+  callback: () => {},
+  instance: 'test',
+  room: 'some-room',
+  expireTime: 10000,
+  refreshTime: 1000,
+});
+
+```
+
+### Methods
+
+- `join()` - Joins the room provided in the constructor args and automatically starts listening for new messages
```

Comment:
Why don't we pass the room name as a parameter to `join`? That way we would be able to more easily implement multi-room support if needed in the future.